flowey/flowey_lib_hvlite/src/_jobs/build_and_run_nextest_vmm_tests.rs

```diff
@@ -213,14 +214,16 @@ impl SimpleFlowNode for Node {
             get_env: v,
         });
 
+        let hyperv_test_deps = ctx.reqv(crate::init_hyperv_tests::Request);
+
         let results = ctx.reqv(|v| crate::build_nextest_vmm_tests::Request {
             profile,
             target,
             build_mode: BuildNextestVmmTestsMode::ImmediatelyRun {
                 nextest_profile,
                 nextest_filter_expr,
                 extra_env,
-                pre_run_deps: Vec::new(),
+                pre_run_deps: vec![hyperv_test_deps],
```

Comment:
Since this is (I believe) the node for running vmm tests locally, it seems like we shouldn't automatically do this machine prep task, which screws with global configuration.

In any case, it's not obvious that we should do it, so I'd leave it out for the moment.